themes/digital.gov/layouts/events/single.html

```diff
@@ -108,73 +106,71 @@ <h1>{{- .Title | markdownify -}}</h1>
 
 
       <section>
-        <div class="grid-container grid-container-desktop">
-          <div class="grid-row tablet-lg:grid-gap-4">
-            <div class="grid-col-12 tablet:grid-col-9">
-              {{/* ZOOM Stage */}}
-              {{- if eq .Params.event_platform "zoom" -}}
-                {{/* If is a Future Event */}}
-                {{- if eq $future_event true -}}
-                  {{- .Render "stage-zoom" -}}
-                {{- end -}}
+        <div class="grid-row tablet-lg:grid-gap-4">
+          <div class="grid-col-12 tablet:grid-col-9">
+            {{/* ZOOM Stage */}}
+            {{- if eq .Params.event_platform "zoom" -}}
+              {{/* If is a Future Event */}}
+              {{- if eq $future_event true -}}
+                {{- .Render "stage-zoom" -}}
               {{- end -}}
+            {{- end -}}
 
-              {{/* Adobe Connect Stage */}}
-              {{- if eq .Params.event_platform "adobe_connect" -}}
-                {{/* If is a Future Event */}}
-                {{- if eq $future_event true -}}
-                  {{- .Render "stage-adobe_connect" -}}
-                {{- end -}}
+            {{/* Adobe Connect Stage */}}
+            {{- if eq .Params.event_platform "adobe_connect" -}}
+              {{/* If is a Future Event */}}
+              {{- if eq $future_event true -}}
+                {{- .Render "stage-adobe_connect" -}}
               {{- end -}}
+            {{- end -}}
 
-              {{/* Google Stage */}}
-              {{- if eq .Params.event_platform "google" -}}
-                {{/* If is a Future Event */}}
-                {{- if eq $future_event true -}}
-                  {{- .Render "stage-google" -}}
-                {{- end -}}
+            {{/* Google Stage */}}
+            {{- if eq .Params.event_platform "google" -}}
+              {{/* If is a Future Event */}}
+              {{- if eq $future_event true -}}
+                {{- .Render "stage-google" -}}
               {{- end -}}
+            {{- end -}}
 
```

Comment:
@ToniBonittoGSA  It looks like we don't use `google` for future events anymore and we only have 20 files that use this option.

This code will only affect future events, we can remove it if we don't use google for event hosting.

Reply:
Lets comment this out for now no?

Reply:
Yes, thanks
cc: @jdotyoon